[2023-01-03T19:11:46.944+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: accidents_pipeline.dashboard manual__2023-01-03T19:05:54.497116+00:00 [queued]>
[2023-01-03T19:11:46.957+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: accidents_pipeline.dashboard manual__2023-01-03T19:05:54.497116+00:00 [queued]>
[2023-01-03T19:11:46.957+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-01-03T19:11:46.958+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2023-01-03T19:11:46.958+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-01-03T19:11:46.976+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): dashboard> on 2023-01-03 19:05:54.497116+00:00
[2023-01-03T19:11:46.982+0000] {standard_task_runner.py:55} INFO - Started process 2119 to run task
[2023-01-03T19:11:46.986+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'accidents_pipeline', 'dashboard', 'manual__2023-01-03T19:05:54.497116+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/Airflow.py', '--cfg-path', '/tmp/tmpjz9i9a94']
[2023-01-03T19:11:46.989+0000] {standard_task_runner.py:83} INFO - Job 55: Subtask dashboard
[2023-01-03T19:11:47.071+0000] {task_command.py:389} INFO - Running <TaskInstance: accidents_pipeline.dashboard manual__2023-01-03T19:05:54.497116+00:00 [running]> on host 321b86f85270
[2023-01-03T19:11:47.165+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=accidents_pipeline
AIRFLOW_CTX_TASK_ID=dashboard
AIRFLOW_CTX_EXECUTION_DATE=2023-01-03T19:05:54.497116+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-03T19:05:54.497116+00:00
[2023-01-03T19:11:51.043+0000] {logging_mixin.py:137} INFO - Dash is running on http://127.0.0.1:8050/
[2023-01-03T19:11:51.043+0000] {dash.py:1968} INFO - Dash is running on http://127.0.0.1:8050/

[2023-01-03T19:11:51.050+0000] {logging_mixin.py:137} INFO -  * Serving Flask app '***'
[2023-01-03T19:11:51.051+0000] {logging_mixin.py:137} INFO -  * Debug mode: off
[2023-01-03T19:11:51.052+0000] {logging_mixin.py:137} WARNING - Address already in use
[2023-01-03T19:11:51.053+0000] {logging_mixin.py:137} WARNING - Port 8050 is in use by another program. Either identify and stop that program, or start the server with a different port.
[2023-01-03T19:11:51.097+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2023-01-03T19:11:51.126+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
